src/classes/Forms.ts

```diff
@@ -1027,6 +1028,61 @@ export default class Forms extends OneBlinkAPI {
     }
     return super.getRequest(`/form-submission-meta/${submissionId}`)
   }
+
+  /**
+   * #### Example
+   *
+   * ```javascript
+   * const parameters = {
+   *  formId: 1,
+   *  submissionId: 'c1f0f27b-4289-4ce5-9807-bf84971991aa'
+   *  submissionEvent: {
+   *    type: 'PDF',
+   *    configuration: {
+   *      toEmail: ['{ELEMENT:Email}'],
+   *      emailSubjectLine: 'Email Subject',
+   *      excludedElementIds: [],
+   *    },
+   *  }
+   * }
+   *
+   *
+   * forms
+   *  .triggerWorkflowEvent(parameters)
+   *  .then(() => {
+   *    // Workflow event has been triggered
+   *  })
+   *  .catch((error) => {
+   *    // Handle error here
+   *  })
+   *
+   * ```
+   *
+   * @param params An object containing all parameters to be passed to the
+   *   function
+   */
+  triggerWorkflowEvent(params: {
```

Comment:
```suggestion
  executeWorkflowEvent(params: {
```

We have used the word execute on the conditional logic properties for a workflow event.